cannon/mipsevm/multithreaded/instrumented_test.go

```diff
@@ -37,7 +37,7 @@ func TestInstrumentedState_MultithreadedProgram(t *testing.T) {
 
 	var stdOutBuf, stdErrBuf bytes.Buffer
 	us := NewInstrumentedState(state, oracle, io.MultiWriter(&stdOutBuf, os.Stdout), io.MultiWriter(&stdErrBuf, os.Stderr), testutil.CreateLogger())
-	for i := 0; i < 1_000_000; i++ {
+	for i := 0; i < 2_000_000; i++ {
```

Comment:
Does this mean running this test on 64-bit requires more instructions?

Reply:
Yes!